src/extract.rs

```diff
@@ -21,38 +33,38 @@ enum Header {
     Size = 0x84,
 }
 
-/// Extracts files from an NDS ROM.
+/// Extracts files from an NDS ROM and extracts them to a given path.
```

Comment:
This doesn't read quite right. Maybe "Extracts files from an NDS ROM to a given path"?